Microsoft.Toolkit.Uwp/Helpers/PrintHelper/PrintHelperOptions.cs

```diff
@@ -144,20 +142,5 @@ private void InitializePossibleDisplayOptions()
                                                                       .Where(p => p.PropertyType == typeof(string))
                                                                       .Select(p => (string)p.GetValue(null));
         }
-
-        private void InitializeDefaultOptions()
-        {
-            Bordering = PrintBordering.Default;
-            MediaSize = PrintMediaSize.Default;
-            MediaType = PrintMediaType.Default;
-            HolePunch = PrintHolePunch.Default;
-            Binding = PrintBinding.Default;
-            Duplex = PrintDuplex.Default;
-            ColorMode = PrintColorMode.Default;
-            Collation = PrintCollation.Default;
-            PrintQuality = PrintQuality.Default;
-            Staple = PrintStaple.Default;
-            Orientation = PrintOrientation.Default;
-        }
```

Comment:
Wonder if this is another bug in disguise...hmmm, not sure what we should do for these cases...

Reply:
It is more or less doing the same as the method above.
Looking at the history, this method has been introduced in the first commit, removed in the second, and re-added in the third. It sit there since and is not used by anyone. 

It should be safe to remove it and keep the reflection based one even if the removed one will be more performant and consume less binary space. It is something that can be verified later.

---
In reply to: [390627828](https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3155#discussion_r390627828) [](ancestors = 390627828)